testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -308,6 +309,7 @@
 PDB_xvf = resource_filename(__name__, 'data/cobrotoxin.pdb')
 TPR_xvf = resource_filename(__name__, 'data/cobrotoxin.tpr')
 TRR_xvf = resource_filename(__name__, 'data/cobrotoxin.trr')
+H5MD_xvf = resource_filename(__name__, 'data/cobrotoxin.h5')
```

Comment:
The file that you add to the repo cannot be named `H5MD_xvf.h5` but must have the name you list here: `cobrotoxin.h5`

Reply:
done! thank you